config/cable.yml

```diff
@@ -1,6 +1,5 @@
 development:
-  adapter: redis
-  url: redis://localhost:6379/1
+  adapter: async
```

Comment:
Not sure why we were using redis in dev but async in staging/prod previously as the adapter for Action Cable.

I think we are only using Action Cable for turbo streams atm.

Note: async isn't recommended for production: https://guides.rubyonrails.org/action_cable_overview.html#async-adapter

So we may want to switch to the [action cable postgres adapter](https://guides.rubyonrails.org/action_cable_overview.html#postgresql-adapter), or to [solid_cable](https://github.com/rails/solid_cable) at some point.